### DISCUSSION TITLE
Move Up and Down List

### DISCUSSION THREAD

**Discussion by jrock2004** [LazyVim/LazyVim#858] [Q&A] [answered]
So in my react test I type `screen.` and I have to hit `Ctrl+space` to being up the list. How do I move up and down the list? I know I can do it with arrow keys, but that is not very vim like. What am I missing?

<img width="473" alt="image" src="https://github.com/LazyVim/LazyVim/assets/655716/ee61b403-fc9b-4465-9447-b79dbef6957d">

Side question: is there a reason I need to hit `Ctrl+space` to open the list in the first place?

**Comment by loiccoyle** [comment, marked as answer]
`ctrl+n` and `ctrl+p`
